src/css/components/InputGroup.scss

```diff
@@ -45,22 +46,40 @@ $form-input-height: $su-xlarge !default;
 
 .InputGroup--leftIcon {
   .Input {
-    padding-left: scaleGrid(5);
+    padding-left: scaleGrid(4);
   }
 
-  .InputGroup-icon {
+  .icon-Left {
     left: 0;
-    border-right: $border-width-small solid $cu-divider--light;
   }
 }
 
 .InputGroup--rightIcon {
   .Input {
-    padding-right: scaleGrid(5);
+    padding-right: scaleGrid(4);
   }
 
-  .InputGroup-icon {
+  &.InputGroup--auxIcon {
+    .Input {
+      padding-right: scaleGrid(8);
+    }
+
+    .icon-Right {
+      right: scaleGrid(3);
+    }
+  }
+
+  .icon-Right {
+    right: 0;
+  }
+}
+
+.InputGroup--auxIcon {
+  .Input {
+    padding-right: scaleGrid(4);
+  }
+
+  .icon-Aux {
```

Comment:
When this is used for `showClear` I think it could use some styles to indicate it is interactive. And maybe it gets another modifier class added for the `showClear` state so it doesn't impact the status colors. Something like:

```
.InputGroup-icon--aux { // same as current
  right: 0;    
}

.InputGroup-icon--clear {
  cursor: pointer;
  transition: color ease 0.2s;
  &:hover {
    color: color-neutral(8);
  }
}
```

Reply:
Great suggestion! This looks so much better!